### DISCUSSION TITLE
osu! Lazer Dislikes a Certain Earphone Models

### DISCUSSION THREAD

**Discussion by Shirou27** [ppy/osu#27490] [Q&A]
When I insert a certain model of earphone and after playing with it, it will causes a crash, if not, a sound will disappear and needed to restart to restore the sound.

I bought another earphone which is a HiFi earphone and I tried it playing with osu! Lazer. It doesn't crash anymore nor getting a notification about getting error in sound driver inside osu! lazer notification bar.

My hypothesis is that either osu! Lazer conflict with something to certain earphone or the earphone itself is faulty and osu! Lazer doesn't like that.

**Operating System:** Android 12
**Phone Model:** Redmi Note 9

**Comment by Rekunan** [comment]
What's the certain model of earphone you had trouble with?

**Reply by Shirou27** [reply]
BAVIN HX816

**Reply by Rekunan** [reply]
Haven't heard anything about that one, and can't find much online. Could you attach your log files? Locations are listed in #15188
